snuba/datasets/processors/spans_processor.py

```diff
@@ -224,165 +181,53 @@ def _process_measurements(
                 )
 
     @staticmethod
-    def _process_module_details(
+    def _process_sentry_tags(
         processed: MutableMapping[str, Any],
-        event_dict: EventDict,
+        span_event: SpanEvent,
     ) -> None:
         """
         TODO: For the top level span belonging to a transaction, we do not know how to fill these
               values yet. For now lets just set them to their default values.
         """
-        processed["module"] = ""
-        processed["action"] = ""
-        processed["domain"] = ""
-        processed["status"] = 0
+        sentry_tags = span_event["sentry_tags"]
```

Comment:
Let's assume all data in `sentry_tags` to be optional for now. But the data model has a few required fields. Can you confirm that we are mandating all required fields here and all optional fields are populated with safe defaults in case they are missing?

Reply:
Added a test with the empty `sentry_tags` and missing optional fields in `test_spans_payloads.py`